designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -685,23 +680,23 @@ internal fun defaultLayoutStyle(): LayoutStyle.Builder {
     builder.align_self = AlignSelf.Auto()
     builder.align_content = AlignContent.FlexStart()
     builder.justify_content = JustifyContent.FlexStart()
-    builder.top = Dimension.Undefined()
-    builder.left = Dimension.Undefined()
-    builder.bottom = Dimension.Undefined()
-    builder.right = Dimension.Undefined()
-    builder.margin = zeroRect()
-    builder.padding = zeroRect()
+    builder.top = Optional.of(newDimensionProtoUndefined())
```

Comment:
Can you make `newDimensionProtoUndefined` return `Optional<>` to avoid having `Optional.of()` everywhere here